### PR TITLE
replace ignore_errors with failed_when

### PR DESCRIPTION
**Pull request by odra**
replacing `ignore_errors` with `failed_when` checks.

### CODE REVIEW

File: evals/roles/che/tasks/uninstall.yml [hunk 1]
```diff
@@ -19,13 +19,13 @@
   name: Retrieve service account secrets
   shell: "oc get sa -o jsonpath='{.items[*].secrets[*].name}' -l app={{ che_app_label }} -n {{ che_namespace }}"
   register: che_sa_cmd
-  ignore_errors: true
+  failed_when: false
```
Comment (david-martin) [inline evals/roles/che/tasks/uninstall.yml]: If the serviceaccount doesn't exist, I think its OK to continue (bad install or user manually removed the sa) as the delete step only runs if this command exits 0.

**Comment by david-martin** [comment]
@odra what is the difference with this change?
Are there different error behaviours that do cause the playbook to fail?

**Comment by odra** [comment]
It doesn't show as much red text in the screen as it was being displayed, since we didn't want to give the impression that the playbook failed for some reason.

**Comment by david-martin** [comment]
@aidenkeating i'm good with these changes if you are

**Pull request merged**